### DISCUSSION TITLE
Welcome to linutil Discussions!

### DISCUSSION THREAD

**Discussion by ChrisTitusTech** [ChrisTitusTech/linutil#185] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jee-mj** [comment]
Hey there! Long time hacker/dev. Did a big oopsie right around when I was 18 like 10 years ago and decided to go anon and stuff.

I found this project interesting because I use NixOS and this is an OS utility for Linux distributions, but I haven't really looked at it much because I've learned that I need to rely on explicit guidelines such as documentation for the best application of myself towards something.

So I'm just watching from the sidelines for now. Got busy again recently, have things to look at for the rest of this year.

I'm happy to see that development is proceeding at a decent pace. Congratulations and thank you to all contributors, including you @ChrisTitusTech.

**Comment by nsnelg17** [comment]
I have a question. On linux, why do app just use the 2.0 audio output and not the settings on 2.1 output to which I set it to. How can you make the apps use the 2.1 output. I like to listen to what I creat or hear different sounds like the bass and make sure the levels are correct for bass and other levels. I like the separation of sounds. Please help.
